README.md

```diff
@@ -1,39 +1,89 @@
-# Role Name
-
-A brief description of the role goes here.
+# zfs-snap-manager
 
+This role installs and configures [zfs-snap-manager](https://github.com/khenderick/zfs-snap-manager), a tool to automatically create, replicate and destroy ZFS snapshots.
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
-
+`python2` has to be installed on the target machine. Obviously, any ZFS filesystems that should be snapshotted must exist.
+The machine that executes the playbook needs to have internet access as well as `git` installed.
```

Comment:
I can't possible know which obscure distribution the executing machine is running. I don't want to add tasks for this, especially as `git` it is a very common tool, which in most cases will be used to clone this repository in the first place.